git/essentials/intro-git/benefits-of-git.md

```diff
@@ -0,0 +1,49 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[Why Git](https://www.atlassian.com/git/tutorials/why-git){website}'
+
+---
+
+# Benefits of Git
+
+---
+## Content
+
+Let's talk a bit about what **Git** brings to the table.
+
+1. Apart from syncing changes, everything can be done locally.
```

Comment:
Hmmm, I feel like this sentence doesn't hold enough info.

How about something like this? 

```suggestion
1. Apart from syncing new changes from other people, everything can be done locally.
```